db/migrate/20150714132601_remove_is_param.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveIsParam < ActiveRecord::Migration
+  def up
+    remove_column :lookup_keys, :is_param
+  end
+
+  def down
+    add_column :lookup_keys, :is_param
```

Comment:
<pre>
==  RemoveIsParam: reverting ==================================================
-- add_column(:lookup_keys, :is_param)
rake aborted!
ArgumentError: wrong number of arguments (2 for 3..4)
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/activerecord-3.2.21/lib/active_record/connection_adapters/sqlite_adapter.rb:398:in `add_column'
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/activerecord-3.2.21/lib/active_record/migration.rb:466:in `block in method_missing'
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/activerecord-3.2.21/lib/active_record/migration.rb:438:in `block in say_with_time'
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/activerecord-3.2.21/lib/active_record/migration.rb:438:in `say_with_time'
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/activerecord-3.2.21/lib/active_record/migration.rb:458:in `method_missing'
/home/dcleal/code/foreman/foreman/db/migrate/20150714132601_remove_is_param.rb:7:in `down'
</pre>